Client/Frontend/Brave Rewards/Ads/AdContentButton.swift

```diff
@@ -55,7 +55,7 @@ class AdContentButton: UIControl {
     stackView.addArrangedSubview(titleLabel)
     stackView.addArrangedSubview(bodyLabel)
 
-    let iconImageView = UIImageView(image: UIImage(imageLiteralResourceName: "bat-small"))
+    let iconImageView = UIImageView(image: UIImage(named: "bat-small", in: .current, compatibleWith: nil)!)
```

Comment:
What do you think of adding a helper method to UIImage to skip setting `compatibleWith: nil` argument?
To call it like `UIImage(named: "bat-small", in: .current)`